### ISSUE TITLE
IllegalArgumentException when creating a new message connecting to the start of an existing execution

### ISSUE THREAD

**Issue by planger** [eclipsesource/papyrus-seqd#354]
```!ENTRY org.eclipse.core.jobs 4 2 2018-09-13 14:07:35.435
!MESSAGE An internal error occurred during: "Computing logical model".
!STACK 0
java.lang.IllegalArgumentException: dependency cycle: [ActionExecutionSpecification{G}(SEQD::Interaction1::Execution1, Shape_Execution_Specification), MessageOccurrenceSpecification(SEQD::Interaction1::<unnamed>)]
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.checkForCycle(EdgeImpl.java:156)
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.<init>(EdgeImpl.java:41)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:98)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:94)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:116)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.lambda$5(GraphComputer.java:260)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseExecutionSpecification(GraphComputer.java:260)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseExecutionSpecification(GraphComputer.java:1)
	at org.eclipse.uml2.uml.util.UMLSwitch.doSwitch(UMLSwitch.java:4899)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:53)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:69)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.compute(GraphComputer.java:89)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.<init>(GraphImpl.java:77)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.getInstance(GraphImpl.java:96)
	at org.eclipse.papyrus.uml.interaction.graph.Graph.compute(Graph.java:179)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.<init>(InteractionModelBuilder.java:44)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.getInstance(InteractionModelBuilder.java:57)
	at org.eclipse.papyrus.uml.interaction.model.MInteraction.getInstance(MInteraction.java:56)
	at org.eclipse.papyrus.uml.interaction.model.MInteraction.getInstance(MInteraction.java:70)
	at org.eclipse.papyrus.uml.interaction.model.internal.view.LogicalModelPage$3.lambda$0(LogicalModelPage.java:200)
	at org.eclipse.papyrus.infra.emf.readonly.PapyrusROTransactionalEditingDomain.runExclusive(PapyrusROTransactionalEditingDomain.java:271)
	at org.eclipse.papyrus.uml.interaction.model.internal.view.LogicalModelPage$3.run(LogicalModelPage.java:200)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.ui 4 4 2018-09-13 14:07:37.730
!MESSAGE An internal error has occurred.
!STACK 0
java.lang.IllegalArgumentException: dependency cycle: [ActionExecutionSpecification{G}(SEQD::Interaction1::Execution1, Shape_Execution_Specification), MessageOccurrenceSpecification(SEQD::Interaction1::<unnamed>)]
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.checkForCycle(EdgeImpl.java:156)
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.<init>(EdgeImpl.java:41)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:98)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:94)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:116)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.lambda$5(GraphComputer.java:260)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseExecutionSpecification(GraphComputer.java:260)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseExecutionSpecification(GraphComputer.java:1)
	at org.eclipse.uml2.uml.util.UMLSwitch.doSwitch(UMLSwitch.java:4899)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:53)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:69)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.compute(GraphComputer.java:89)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.<init>(GraphImpl.java:77)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.getInstance(GraphImpl.java:96)
	at org.eclipse.papyrus.uml.interaction.graph.Graph.compute(Graph.java:179)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.<init>(InteractionModelBuilder.java:44)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.getInstance(InteractionModelBuilder.java:57)
	at org.eclipse.papyrus.uml.interaction.model.MInteraction.getInstance(MInteraction.java:56)
	at org.eclipse.papyrus.uml.diagram.sequence.runtime.internal.edit.policies.LogicalModelCreationEditPolicy.getCreateElementAndViewCommand(LogicalModelCreationEditPolicy.java:50)
	at org.eclipse.gmf.runtime.diagram.ui.editpolicies.CreationEditPolicy.getCommand(CreationEditPolicy.java:87)
	at org.eclipse.papyrus.uml.diagram.sequence.runtime.internal.edit.policies.InteractionCreationEditPolicy.getCommand(InteractionCreationEditPolicy.java:41)
	at org.eclipse.gef.editparts.AbstractEditPart.getCommand(AbstractEditPart.java:502)
	at org.eclipse.gmf.runtime.diagram.ui.editparts.GraphicalEditPart.access$1(GraphicalEditPart.java:1)
	at org.eclipse.gmf.runtime.diagram.ui.editparts.GraphicalEditPart$1.run(GraphicalEditPart.java:482)
	at org.eclipse.papyrus.infra.emf.readonly.PapyrusROTransactionalEditingDomain.runExclusive(PapyrusROTransactionalEditingDomain.java:271)
	at org.eclipse.gmf.runtime.diagram.ui.editparts.GraphicalEditPart.getCommand(GraphicalEditPart.java:477)
	at org.eclipse.papyrus.uml.diagram.sequence.runtime.internal.edit.policies.SequenceDiagramPopupBarEditPolicy$DefaultTool.getCommand(SequenceDiagramPopupBarEditPolicy.java:191)
	at org.eclipse.gmf.runtime.diagram.ui.tools.AbstractPopupBarTool.isCommandEnabled(AbstractPopupBarTool.java:96)
	at org.eclipse.papyrus.infra.gmfdiag.common.editpolicies.PapyrusPopupBarEditPolicy.addPopupBarDescriptor(PapyrusPopupBarEditPolicy.java:325)
	at org.eclipse.papyrus.infra.gmfdiag.common.editpolicies.PapyrusPopupBarEditPolicy.addPopupBarDescriptor(PapyrusPopupBarEditPolicy.java:347)
	at org.eclipse.papyrus.infra.gmfdiag.common.editpolicies.PapyrusPopupBarEditPolicy.addPopupBarDescriptor(PapyrusPopupBarEditPolicy.java:360)
	at org.eclipse.papyrus.infra.gmfdiag.common.editpolicies.PapyrusPopupBarEditPolicy.populatePopupBars(PapyrusPopupBarEditPolicy.java:198)
	at org.eclipse.papyrus.infra.gmfdiag.common.editpolicies.PapyrusPopupBarEditPolicy.showDiagramAssistant(PapyrusPopupBarEditPolicy.java:509)
	at org.eclipse.papyrus.uml.diagram.sequence.runtime.internal.edit.policies.SequenceDiagramPopupBarEditPolicy.showDiagramAssistant(SequenceDiagramPopupBarEditPolicy.java:56)
	at org.eclipse.gmf.runtime.diagram.ui.editpolicies.DiagramAssistantEditPolicy$ShowDiagramAssistantJob.runInUIThread(DiagramAssistantEditPolicy.java:98)
	at org.eclipse.ui.progress.UIJob.lambda$0(UIJob.java:95)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:37)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:182)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4577)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:4186)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1150)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1039)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:153)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:680)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:594)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:148)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:151)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:653)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:590)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1499)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1472)``'

**Comment by planger** [comment]
It seems, this has been introduced with 66011de8b5573e3a99d5bc0e82a3d03e620cc14e

**Comment by planger** [comment]
Closed via #355

**Comment by planger** [comment]
I encountered another occurrence of this exception when creating a delete message. It can be reproduced when adding a delete message from `Lifeline1` to `Lifeline2` in the interaction of [IAE-DeleteMessage.zip](https://github.com/eclipsesource/papyrus-seqd/files/2387635/IAE-DeleteMessage.zip).

The exception that occurs is as follows:
```
!ENTRY org.eclipse.core.jobs 4 2 2018-09-17 09:31:20.876
!MESSAGE An internal error occurred during: "Computing logical model".
!STACK 0
java.lang.IllegalArgumentException: dependency cycle: [Lifeline{G}(SEQD2::Interaction1::Lifeline2, Shape_Lifeline_Header), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), Message(SEQD2::Interaction1::<unnamed>, Edge_Message), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), Message(SEQD2::Interaction1::<unnamed>, Edge_Message)]
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.checkForCycle(EdgeImpl.java:156)
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.<init>(EdgeImpl.java:41)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:98)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:94)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.lambda$0(GraphComputer.java:112)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.DistinctOps$1$2.accept(DistinctOps.java:175)
	at java.util.Iterator.forEachRemaining(Iterator.java:116)
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:112)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:124)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseMessage(GraphComputer.java:209)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseMessage(GraphComputer.java:1)
	at org.eclipse.uml2.uml.util.UMLSwitch.doSwitch(UMLSwitch.java:4667)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:53)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:69)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.compute(GraphComputer.java:89)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.<init>(GraphImpl.java:77)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.getInstance(GraphImpl.java:96)
	at org.eclipse.papyrus.uml.interaction.graph.Graph.compute(Graph.java:179)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.<init>(InteractionModelBuilder.java:44)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.getInstance(InteractionModelBuilder.java:57)
	at org.eclipse.papyrus.uml.interaction.model.MInteraction.getInstance(MInteraction.java:56)
	at org.eclipse.papyrus.uml.interaction.model.MInteraction.getInstance(MInteraction.java:70)
	at org.eclipse.papyrus.uml.interaction.model.internal.view.LogicalModelPage$3.lambda$0(LogicalModelPage.java:200)
	at org.eclipse.papyrus.infra.emf.readonly.PapyrusROTransactionalEditingDomain.runExclusive(PapyrusROTransactionalEditingDomain.java:271)
	at org.eclipse.papyrus.uml.interaction.model.internal.view.LogicalModelPage$3.run(LogicalModelPage.java:200)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.ui 4 0 2018-09-17 09:31:20.878
!MESSAGE Unhandled event loop exception
!STACK 0
java.lang.IllegalArgumentException: dependency cycle: [Lifeline{G}(SEQD2::Interaction1::Lifeline2, Shape_Lifeline_Header), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), Message(SEQD2::Interaction1::<unnamed>, Edge_Message), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), MessageOccurrenceSpecification(SEQD2::Interaction1::<unnamed>), Message(SEQD2::Interaction1::<unnamed>, Edge_Message)]
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.checkForCycle(EdgeImpl.java:156)
	at org.eclipse.papyrus.uml.interaction.internal.graph.EdgeImpl.<init>(EdgeImpl.java:41)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:98)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:94)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.lambda$0(GraphComputer.java:112)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.DistinctOps$1$2.accept(DistinctOps.java:175)
	at java.util.Iterator.forEachRemaining(Iterator.java:116)
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:112)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.edge(GraphComputer.java:124)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseMessage(GraphComputer.java:209)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer$EdgeFactory.caseMessage(GraphComputer.java:1)
	at org.eclipse.uml2.uml.util.UMLSwitch.doSwitch(UMLSwitch.java:4667)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:53)
	at org.eclipse.emf.ecore.util.Switch.doSwitch(Switch.java:69)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphComputer.compute(GraphComputer.java:89)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.<init>(GraphImpl.java:77)
	at org.eclipse.papyrus.uml.interaction.internal.graph.GraphImpl.getInstance(GraphImpl.java:96)
	at org.eclipse.papyrus.uml.interaction.graph.Graph.compute(Graph.java:179)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.<init>(InteractionModelBuilder.java:44)
	at org.eclipse.papyrus.uml.interaction.internal.model.impl.InteractionModelBuilder.getInstance(InteractionModelBuilder.java:57)
	at org.eclipse.papyrus.uml.interaction.model.MInteraction.getInstance(MInteraction.java:56)
	at org.eclipse.papyrus.uml.interaction.model.MInteraction.getInstance(MInteraction.java:70)
	at org.eclipse.papyrus.uml.diagram.sequence.runtime.internal.edit.parts.LifelineBodyEditPart.getMInteraction(LifelineBodyEditPart.java:129)
	at org.eclipse.papyrus.uml.diagram.sequence.runtime.internal.edit.parts.LifelineBodyEditPart.computeLifelineHeight(LifelineBodyEditPart.java:104)
	at org.eclipse.papyrus.uml.diagram.sequence.runtime.internal.edit.parts.LifelineBodyEditPart.refreshBounds(LifelineBodyEditPart.java:91)
	at org.eclipse.gmf.runtime.diagram.ui.editparts.ShapeEditPart.refreshVisuals(ShapeEditPart.java:205)
	at org.eclipse.gef.editparts.AbstractEditPart.refresh(AbstractEditPart.java:725)
	at org.eclipse.gef.editparts.AbstractGraphicalEditPart.refresh(AbstractGraphicalEditPart.java:644)
	at org.eclipse.gmf.runtime.diagram.ui.editparts.GraphicalEditPart.access$3(GraphicalEditPart.java:1)
	at org.eclipse.gmf.runtime.diagram.ui.editparts.GraphicalEditPart$3.run(GraphicalEditPart.java:861)
	at org.eclipse.papyrus.infra.emf.readonly.PapyrusROTransactionalEditingDomain.runExclusive(PapyrusROTransactionalEditingDomain.java:271)
	at org.eclipse.gmf.runtime.diagram.ui.editparts.GraphicalEditPart.refresh(GraphicalEditPart.java:851)
	at org.eclipse.papyrus.infra.gmfdiag.common.helper.DiagramHelper.refresh(DiagramHelper.java:55)
	at org.eclipse.papyrus.infra.gmfdiag.common.helper.DiagramHelper.refresh(DiagramHelper.java:64)
	at org.eclipse.papyrus.infra.gmfdiag.common.helper.DiagramHelper.refresh(DiagramHelper.java:64)
	at org.eclipse.papyrus.infra.gmfdiag.common.helper.DiagramHelper.refresh(DiagramHelper.java:64)
	at org.eclipse.papyrus.infra.gmfdiag.common.helper.DiagramHelper.refresh(DiagramHelper.java:84)
	at org.eclipse.papyrus.infra.gmfdiag.common.helper.DiagramHelper$1.run(DiagramHelper.java:149)
	at org.eclipse.papyrus.infra.ui.util.UIUtil$UIExecutorService$RunnableWrapper.run(UIUtil.java:536)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:37)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:182)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4577)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:4186)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1150)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1039)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:153)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:680)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:594)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:148)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:151)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:653)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:590)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1499)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1472)
```
